### PR TITLE
New Extension: Library Genesis

### PR DESCRIPTION
**Pull request by yz3440**
## Description

An extension that allows searching books on [Library Genesis](https://libgen.is) and quickly opening the link in the browser, using the [libgen.js](https://github.com/dunn/libgen.js/) API.

## Screencast

Screencast of Library Genesis Extension

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/library-genesis/metadata/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Library Genesis Changelog
```
Comment (danielsequeira) [inline extensions/library-genesis/metadata/CHANGELOG.md:1]: seems like a duplicated file from your already existing `CHANGELOG.md` in the project root, you can delete this one or move it to root and overwrite the other one

**Review by danielsequeira — CHANGES_REQUESTED** [review]
Hi @yz3440 👋 

Thank you so much for your PR.

I went through the code and left a few comments, nothing major.

Besides the comments you can also delete the screencasts folder and its content, since you don't really need it.

File: extensions/library-genesis/src/components/book-item.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { List, ActionPanel, Action } from "@raycast/api";
+import { BookEntry } from "../types";
+import { formatBytes } from "../utils/common-utils";
+
+export function BookItem(props: { book: BookEntry }, key: number) {
+  const { book } = props;
+  const markdown = `
+  <img src="${book.coverUrl}" alt="cover" height="180"/>
+
```
[the remark this answers is not in the document]
Comment (yz3440) [reply extensions/library-genesis/src/components/book-item.tsx:8]: Sharp eye! It was a mistake.

**Comment by danielsequeira** [comment]
@yz3440 all looks good, the only thing missing is removing the screencasts folder, since it is redundant considering the metadata one. Besides that you can change the date in the changelog to today's date and then I can go ahead and merge it 🙂

**Comment by yz3440** [comment]
@danielsequeira thanks! I had kept the screencasts folder because the image on top of this PR references the URL. I did it anyway. Now that I've removed it, the image src shows 404.

File: extensions/library-genesis/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Library Genesis
+
+Search books on Library Genesis and view them.
+
+![Screencast](./screencasts/library-genesis-1.png)
```
Comment (danielsequeira) [inline extensions/library-genesis/README.md:5]: Sorry @yz3440 I just noticed that you used the image in the screencasts folder in your README. for images used in the README you should place them in the `assets` folder. So one last thing to would be to add this image to your existing `assets` folder

**Review by danielsequeira — APPROVED** [review]

**Pull request merged**